cmake/modules/extensions.cmake

```diff
@@ -3332,6 +3332,49 @@ macro(zephyr_linker_memory_ifdef feature_toggle)
   endif()
 endmacro()
 
+# Usage:
+#   zephyr_linker_dts_section_from_path(PATH <path>)
+#
+# Zephyr linker devicetree memory section from path.
+#
+# This function specifies a memory section for the platform in use based on its
```

Comment:
I think `output section` is a more correct term than `memory section` wrt linkers, wouldn't you agree.

Reply:
agree